samples/tfm_integration/tfm_secure_partition/CMakeLists.txt

```
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13.1)
```

Please bump version:
```suggestion
cmake_minimum_required(VERSION 3.20)
```